runtime/src/append_vec.rs

```diff
@@ -28,6 +28,8 @@ use {
     },
 };
 
+pub mod test_utils;
```

Comment:
I'm not too familiar with the right way to do this code organization.
Does the pub mod go here or should it be in lib?

Reply:
Putting it here preserves the current usage. So it's namespaced like this (both before this PR, and with this PR):
```
runtime::append_vec::test_utils
```

If this line were in `runtime/lib.rs` then the namespacing would change to:
```
runtime::test_utils
```
which I don't think is right.

Reply:
not a big deal at any rate